engine/execution/testutil/fixtures.go

```diff
@@ -215,7 +215,7 @@ func RootBootstrappedLedger(vm *fvm.VirtualMachine, ctx fvm.Context) *state.MapL
 
 	_ = vm.Run(
 		ctx,
-		fvm.Bootstrap(unittest.ServiceAccountPublicKey, unittest.GenesisTokenSupply),
+		fvm.Bootstrap(unittest.ServiceAccountPublicKey, fvm.WithInitialTokenSupply(unittest.GenesisTokenSupply)),
```

Comment:
Same as above, the nested expressions in the argument list are hard to read, maybe introduce a variable for `fvm.Bootstrap` and put its arguments on separate lines